acquire/acquire.py

```diff
@@ -261,17 +261,15 @@ class Sys(Module):
 
     @classmethod
     def _run(cls, target: Target, collector: Collector):
-        if not Path("/sys").exists():
+        if not str(target.path) == "local":
```

Comment:
Just a note, this method (and the way `/proc` and `/sys` are currently being mounted) won't work in the future with a remote agent or targetd target. Perhaps mounting of `/proc` and `/sys` should be moved to the `LocalLoader` instead.

Reply:
I made a ticket for this.